libunwind/test/aarch64_vg_unwind.pass.cpp

```diff
@@ -6,7 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: linux && target={{aarch64-.+}}
+// REQUIRES: target-aarch64
+// UNSUPPORTED: target={{.*-w64-windows-gnu}}
```

Comment:
I would suggest to not use overly narrow triples for these `UNSUPPORTED` markings; in particular, there's no point in including the vendor field (`w64`) in the triple, just doing `target={{.*-windows-gnu}}` would be better. And if we don't strictly need to make the distinction, I think it'd also be better to cover any windows environment (i.e. `target={{.*windows-.*}}` - it'd be highly unlikely that these tests would be passing in an MSVC style environment but failing in a mingw environment.